src/components/MoneyRequestHeaderStatusBar.tsx

```diff
@@ -19,9 +20,13 @@ function MoneyRequestHeaderStatusBar({title, description, shouldShowBorderBottom
     const borderBottomStyle = shouldShowBorderBottom ? styles.borderBottom : {};
     return (
         <View style={[styles.dFlex, styles.flexRow, styles.alignItemsCenter, styles.flexGrow1, styles.overflowHidden, styles.ph5, styles.pb3, borderBottomStyle]}>
-            <View style={[styles.moneyRequestHeaderStatusBarBadge]}>
-                <Text style={[styles.textStrong, styles.textMicroBold]}>{title}</Text>
-            </View>
+            {typeof title === 'string' ? (
```

Comment:
There's another use case for this component which is using a text with a styled container and we want to use an icon without a background in there instead. This approach gives us more freedom, because we can customize the content to our linking.